### PR TITLE
Create charmhub beta

### PR DESCRIPTION
**Pull request by codeEmpress1**
## Done
- create charmhub /beta 
## How to QA
- Go to : 
https://charmhub-io-1517.demos.haus/beta/store
https://charmhub-io-1517.demos.haus/beta/charms
https://charmhub-io-1517.demos.haus/beta/bundles
https://charmhub-io-1517.demos.haus/beta/login
https://charmhub-io-1517.demos.haus/beta/logout
https://charmhub-io-1517.demos.haus/beta/account.json

## Issue / Card
Fixes #

### CODE REVIEW

**Comment by codeEmpress1** [comment]
> 

Yes! that is happening because I added github link for store base to requirements, I will remove it and update. However, QA will only be done locally for now.

**Review by Lukewh — APPROVED** [review]
:+1: LGTM

File: requirements.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ canonicalwebteam.image-template==1.3.1
 canonicalwebteam.store-api==4.0.0
 canonicalwebteam.docstring-extractor==1.1.2
 Flask-WTF==1.0.1
-humanize==4.4.0
+humanize
```
Comment (Lukewh) [inline requirements.txt:9]: we should pin the version

File: requirements.txt [hunk 2]
```diff
@@ -16,3 +16,4 @@ ruamel.yaml==0.17.21
 PyGithub==1.57
 black==22.12.0
 flake8==6.0.0
+-e git+https://git@github.com/codeEmpress1/canonicalwebteam.store-base.git@2cfb9162a2521e6aa21461a76bb48478bf404d18#egg=canonicalwebteam_store_base
```
Comment (Lukewh) [inline requirements.txt:19]: does the image build if you remove the `-e` so it's not editable?

Comment (codeEmpress1) [reply requirements.txt]: I removed the -e and it still doesn't build.  It couldn't install store base from git

**Pull request merged**